javascript/package.json

```diff
@@ -4,9 +4,6 @@
     "devDependencies": {
         "jest": "^26.6.3"
     },
-    "jest": {
-        "testRegex": "./test/.*.js$"
```

Comment:
This suggests Jest automatically treats files ending `.test.js` as tests? Nice! TIL.

Reply:
Yep it does 👍 
https://jestjs.io/docs/configuration#testregex-string--arraystring